tests/test_artifacts.py

```diff
@@ -0,0 +1,88 @@
+"""
+Testing protocols for the Lens package. Tested functionalities:
+    1. General inits
+    2. Individual evaluator runs
+"""
+
+from abc import ABC
+
+import pytest
+import pandas as pd
+import numpy as np
+
+from credoai.artifacts import ComparisonData, ComparisonModel, TabularData
+from credoai.artifacts import DummyClassifier
+
+from credoai.datasets import fetch_censusincome, fetch_credit_model
+
+from credoai.lens import Lens
+
+from credoai.utils import ValidationError
+
+
+def test_base_data_property():
+    """
+    tests call to Base.data property
+    """
+    X_test, y_test, _, _ = fetch_credit_model()
+
+    credo_data = TabularData(name="test_data_property", X=X_test, y=y_test)
+    pytest.assume(not print(credo_data.data))
```

Comment:
The point is explicitly to cover the `.data` property of BaseData (through TabularData).

It's a not because the print function doesn't return anything. I could just do `pytest.assume(credo_data.data)`